### PR TITLE
Add missing Polish translations

### PR DESCRIPTION
**Pull request by HubertStefanski**
**When merged this pull request will:**
- Add missing Polish translations

### CODE REVIEW

**Comment by HubertStefanski** [comment]
@PabstMirror seems like the issue lies on CI/CD? `can't resolve 'github.com'` I'm not sure how to interact with these tools, is there a way to retest? the build shouldn't fail afaict

**Comment by PabstMirror** [comment]
```
##[error]fatal: unable to access 'https://github.com/CBATeam/CBA_A3/': Could not resolve host: github.com
```
weird error, but I forced a re-run and it's working now

**Comment by HubertStefanski** [comment]
> ```
> ##[error]fatal: unable to access 'https://github.com/CBATeam/CBA_A3/': Could not resolve host: github.com
> ```
> 
> weird error, but I forced a re-run and it's working now

Github was down for a brief period, that would explain the fail

File: addons/disposable/stringtable.xml [hunk 1]
```diff
@@ -60,6 +65,7 @@
             <Chinese>只有AI才會丟棄使用過的一次性發射器</Chinese>
             <French>Seules les unités IA lâchent leur lanceur après emploi.</French>
             <Russian>Только ИИ выбрасывает использованный одноразовый гранатомёт.</Russian>
+            <Polish>Tylko AI wyrzuca jednorazową wyrzutnie</Polish>
```
Comment (veteran29) [inline addons/disposable/stringtable.xml:68]:
```suggestion
            <Polish>Tylko AI wyrzuca wyrzutnie jednorazową.</Polish>
```

File: addons/disposable/stringtable.xml [hunk 2]
```diff
@@ -40,6 +43,7 @@
             <Chinese>不要自動丟棄一次性使用之發射器</Chinese>
             <French>Ne lâche jamais automatiquement le lanceur usagé.</French>
             <Russian>Не выбрасывать автоматически использованный одноразовый гранатомёт.</Russian>
+            <Polish>Nie wyrzucaj automatycznie wyrzutni</Polish>
```
Comment (veteran29) [inline addons/disposable/stringtable.xml:46]:
```suggestion
            <Polish>Nie wyrzucaj automatycznie wyrzutni.</Polish>
```

File: addons/disposable/stringtable.xml [hunk 3]
```diff
@@ -80,6 +87,7 @@
             <Chinese>當切換別的武器時自動丟棄一次性發射器。</Chinese>
             <French>Lâche automatiquement le lanceur usagé dès qu'une autre arme est sélectionnée.</French>
             <Russian>Автоматически выбрасывать использованный одноразовый гранатомёт, как только будет выбрано другие оружие.</Russian>
+            <Polish>Automatycznie wyrzucaj jednorazową wyrzutnie gdy tylko zostanie wybrana kolejna broń</Polish>
```
Comment (veteran29) [inline addons/disposable/stringtable.xml:90]:
```suggestion
            <Polish>Automatycznie wyrzucaj wyrzutnie jednorazową gdy tylko zostanie wybrana kolejna broń.</Polish>
```

File: addons/disposable/stringtable.xml [hunk 4]
```diff
@@ -100,6 +109,7 @@
             <Chinese>啟用時，將會把載具或容器裡面給於一次性發射器的彈藥換成一次性發射器本身，並且使其只能使用一次。如果關閉，一次性發射器即可再度裝填</Chinese>
             <French>Si activé, les lanceurs à usage unique ne peuvent être utilisés qu'une seule fois. Les munitions pour lanceurs à usage unique situés dans les conteneurs et les véhicules seront remplacées par des lanceurs à usage unique chargés. Si désactivé, les lanceurs à usage unique peuvent être rechargés après emploi.</French>
             <Russian>Если включено, одноразовые гранатомёты могут быть использованы лишь один раз. Боеприпасы для них в контейнерах и технике будут заменены на соответствующие им заряженные одноразовые гранатомёты. Если выключено, одноразовые гранатомёты могут быть перезаряжены и использованы повторно.</Russian>
+            <Polish>Gdy ta opcja jest włączona wyrzutnie jednorazowe mogą być użyte tylko raz, Amunicja do wyrzutni jednorazowych dostępna w kontenerach i pojazdach zostanie zamieniona w załadowaną wyrzutnie jednorazową. Jeżeli ta opcja jest wyłączona, wtedy wyrzutnie jednorazowe nie zostaną przeładowane po użyciu </Polish>
```
Comment (veteran29) [inline addons/disposable/stringtable.xml:112]:
```suggestion
            <Polish>Gdy ta opcja jest włączona wyrzutnie jednorazowe mogą być użyte tylko raz, Amunicja do wyrzutni jednorazowych dostępna w kontenerach i pojazdach zostanie zamieniona w załadowaną wyrzutnie jednorazową. Jeżeli ta opcja jest wyłączona, wtedy wyrzutnie jednorazowe nie zostaną przeładowane po użyciu.</Polish>
```

**Pull request merged**